_pages/security/dynamic-scanning.md

```diff
@@ -5,29 +5,58 @@ parent: Security
 
 [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is the tool we use for doing automated vulnerability scanning.
 
+<img src="../../assets/images/zap_home.png" alt="zap home screen"/>
```

Comment:
Ditto for the others that don't need anything besides `src` and `alt` attributes.